iulianIachimov/src/main/java/dataCollectionTask/Models/AbstractOrder.java

```diff
@@ -0,0 +1,96 @@
+package dataCollectionTask.Models;
+
+import dataCollectionTask.interfaces.Order;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public abstract class AbstractOrder implements Order {
+  private final UUID id;
+  private final LocalDateTime date;
+  private List<Item> items;
```

Comment:
Please change private to protected

Reply:
I am not sure if it is the case to do this move as per conditions we have - Make all fields `private`. I suppose it is a subject to be discussed tomorrow at Q&A Session